issuer/tests/issue_credentials.rs

```diff
@@ -400,16 +402,120 @@ fn should_prepare_verfied_member_credential_for_authorized_principal() {
     }
 }
 
+#[test]
+fn should_prepare_verified_age_vc_for_weaker_claim() {
```

Comment:
What does "weaker_claim" mean?

Reply:
It is weaker as the lower bound requested for the VC implied by the actual age (if a user is 21, his/her age is at least 18).  I added a comment.